src/drivers/model/src/lib.rs

```diff
@@ -17,23 +17,37 @@ pub trait Driver {
     fn pread(&self, data: &mut [u8], pos: usize) -> Result<usize>;
     /// Positional write. Returns number of bytes written.
     fn pwrite(&mut self, data: &[u8], pos: usize) -> Result<usize>;
+    /// Control the device.
+    /// The data is an arbitrary string.
+    /// This function should not be called directly; rather, use the ctl macro,
+    /// which ensures that the parameter is valid. The ctl function is allowed
+    /// to ignore the request. It will return an error only in the event
+    /// of a real error. The result should included any disambuguiting information
```

Comment:
s/included/include/
s/disambuguiting/disambiguating/

Reply:
It doesn't feel right to me. But I think you're right. I am going to close this.